### DISCUSSION TITLE
qemu archlinux not booting on replit bash

### DISCUSSION THREAD

**Discussion by CorruptionVR** [copy/v86#939] [Q&A] [answered]
hey, i followed the instructions on archlinux.md
```sh
# fetch archlinux32 installer
wget https://mirror.archlinux32.org/archisos/archlinux32-2021.12.01-i686.iso

# Create a 10 gigabyte disk image. If you intend to pacstrap only 'base' then 1.5G should be fine also.
qemu-img create arch.img 10G

# Follow the normal installation process (you can add accel=kvm if your system supports it to speed up the installation)
qemu-system-x86_64 -m 256 -drive file=arch.img,format=raw -cdrom archlinux32-2021.12.01-i686.iso
```
and qemu loads, but i get this
<img width="734" alt="Screen Shot 2023-10-06 at 10 39 02 AM" src="https://github.com/copy/v86/assets/78228404/e9a1fb2e-1100-4d1e-8a5d-92b7f16c78b7">

how do i make this work?
I copy and pasted the commands into the shell

**Comment by copy** [comment, marked as answer]
Try `-m 512`.

**Reply by CorruptionVR** [reply]
that worked. thank you.

**Reply by CorruptionVR** [reply]
also, if I click "Get CD-ROM image" in v86, will it download the image with the progress I made, or will it only give me the original file that I uploaded?

**Reply by CorruptionVR** [reply]
is qemu supposed to kill itself after a while?
